### PR TITLE
Add in setupw, tcp, sst, changes for satwinds, pbl calculation, and misc ush changes

### PR DESCRIPTION
**Pull request by CoryMartin-NOAA**

### CODE REVIEW

File: src/gsi/guess_grids.F90 [hunk 1]
```diff
@@ -1718,6 +1719,95 @@ subroutine load_gsdpbl_hgt(mype)
     return
   end subroutine load_gsdpbl_hgt
 
+!>>emily
```
Comment (ADCollard) [inline src/gsi/guess_grids.F90:1722]: Remove

File: src/gsi/guess_grids.F90 [hunk 2]
```diff
@@ -1718,6 +1719,95 @@ subroutine load_gsdpbl_hgt(mype)
...
+             d=pbl_height(i,j,jj) - k
+             pbl_height(i,j,jj) = pbk(k) * (one-d) + pbk(k+1) * d
+          end do
+       end do
+    end do
+666 format(a24,2x,i6,2x,3(f12.5,2x))
+
+    return
+  end subroutine load_pbl_hgt
+!<<emily
```
Comment (ADCollard) [inline src/gsi/guess_grids.F90:1810]: Remove

File: src/gsi/guess_grids.F90 (hunk 1 above)
Comment (CoryMartin-NOAA) [reply src/gsi/guess_grids.F90:1722]: @ADCollard not sure if this PR is still relevant given that it is 2 years old :-)

Comment (ADCollard) [reply src/gsi/guess_grids.F90:1722]: Oops!  Was poking around for something else and came across this and didn't see the date.  Maybe remove?

**Pull request closed without merging**